readme.md

```diff
@@ -107,14 +114,21 @@ In this microbenchmark we compare two cases:
 
 * serializing an object after an object's property has been set to `undefined`
 * serializing an object after `delete` has been used to remove an object's property
+* serializing an object after `delete` has been used to remove an object's property that
+  was the last property added.
 
 **Code:** <https://github.com/davidmarkclements/v8-perf/blob/master/bench/property-removal.js>
 
 ![](graphs/property-removal-bar.png)
 
-In V8 6.1 (not yet used in any Node releases), operations on an object with a deleted property gets *extremeley* fast,
-faster, even, than setting to `undefined`. This is excellent news because now we can do
-the right thing *and* it's the fastest option.
+In V8 6.0 and 6.1 (not yet used in any Node releases), deleting the last property added to an object hit a fast path in Turbofan,
+and thus it is faster, even, than setting to `undefined`. This is excellent news, as it shows that the V8 team is working
+towards improving the performance of `delete`. Even in latest V8 releases, the `delete` operator might
+still result in massive slowdown, so we still recommend to avoid using it.
+
+_Edit: we incorrectly reported that you could start using delete in
+future Node.js releases, this was corrected in a later edition of the
+post._
```

Comment:
Edit: in a previous version of the post we concluded that `delete` could and should be used 
in future Node.js releases. However <person on v8 team> let us know that our benchmarks only triggered the last property accessed case. Thanks <person on v8 team>!